src/SFA.DAS.AssessorService.Web/Validators/Apply/ApplyStandardConfirmViewModelValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class ApplyStandardConfirmViewModelValidator : AbstractValidator<ApplyStandardConfirmViewModel>
+    {
+        public ApplyStandardConfirmViewModelValidator()
+        {
+            RuleFor(vm => vm.IsConfirmed)
+                .NotEmpty()
```

Comment:
Not sure how this could be empty -- isn't this a non-nullable boolean?

Reply:
This does work as it compares to the default value of the type e.g. false - however better to make that explicit so done that

Reply:
Hmm, that's some unintuitive behaviour on the library's part considering that empty and false are two very different things